packages/utils/src/lib/reports/generate-md-report.ts

```diff
@@ -276,30 +312,32 @@ function reportToAboutSection(report: ScoredReport): string {
   );
 }
 
-function getDocsAndDescription({
+// @TODO extract `Pick<AuditReport, 'docsUrl' | 'description'>` to a reusable schema and type
+export function getDocsAndDescription({
   docsUrl,
   description,
-}: AuditReport | CategoryConfig): string {
+}: Pick<AuditReport, 'docsUrl' | 'description'>): string {
+  const endingNewLine = NEW_LINE + NEW_LINE;
   if (docsUrl) {
     const docsLink = link(docsUrl, '📖 Docs');
     if (!description) {
-      return docsLink + NEW_LINE + NEW_LINE;
+      return docsLink + endingNewLine;
     }
+    // @TODO introduce NEW_LINE at the end of a code block
     if (description.endsWith('```')) {
       // when description ends in code block, link must be moved to next paragraph
-      return description + NEW_LINE + NEW_LINE + docsLink + NEW_LINE + NEW_LINE;
+      return description + NEW_LINE + NEW_LINE + docsLink + endingNewLine;
```

Comment:
Missed one:

```suggestion
      return description + endingNewLine + docsLink + endingNewLine;
```

Reply:
I treated it as "not the end of the line" to just get tests passing with unchanged code. Will refactor more as a couple of bugs popped up during testing.

Reply:
I refactored the usage of NEW_LINE completely in reports.